snappi_ixnetwork/device/base.py

```diff
@@ -122,6 +122,19 @@ def configure_multivalues(self, snappi_obj, ixn_obj, attr_map):
                 ))
             ixn_obj[ixn_attr] = self.multivalue(value)
 
+    def configure_multivalues_with_choice(self, snappi_obj, ixn_obj, attr_map):
+        """attr_map contains snappi_key : ixn_key/ ixn_info in dict format"""
+        self.logger.debug("configuring multivalues:")
+        for snappi_attr, ixn_map in attr_map.items():
+            ixn_attr = ixn_map
```

Comment:
is it require to store `ixn_map` to `ixn_attr`. I am ok if it will require for some other use case

Reply:
yeah..just want this if we get any other property with choice in future